### DISCUSSION TITLE
Please help me to suggest some interesting movies :D

### DISCUSSION THREAD

**Discussion by phuongnaist** [NAIST-SE/AttractingContributors#36] [General]
I need help with my movies list.

To help you, please look at this list
https://www.imdb.com/

Come to my fork:
https://github.com/phuongnaist/AttractingContributors-Movies

Add to my README file.

I look forward to your contributions <3

**Comment by vittunyuta-aom** [comment]
added!! please help me https://github.com/aommoaGitHub/SuperFavoriteSongs

**Comment by tattiyaS** [comment]
Added a movie!

**Comment by noppadol-assava** [comment]
I commit pull request on your repo.
Please help me adding Food into the list.
https://github.com/NoppadolAssava/AttractingContributors---Food

**Comment by 278Mt** [comment]
Added my favorite movie!
And please help me https://github.com/278Mt/AttractingContributors
thx;-)

**Comment by wowry** [comment]
Added, and please help me...
https://github.com/wowry/AttractingContributors-VideoGames

**Comment by manzoku23** [comment]
Added my favorite one.
Please help me
https://github.com/Exe-dev/AttractingContributors-POKEMON
